backend/danswer/indexing/embedder.py

```diff
@@ -69,41 +76,27 @@ def __init__(
 
     def embed_chunks(
         self,
-        chunks: list[DocAwareChunk],
-        enable_mini_chunk: bool = ENABLE_MINI_CHUNK,
+        chunks_with_texts: list[TextChunk],
     ) -> list[IndexChunk]:
-        # Cache the Title embeddings to only have to do it once
-        title_embed_dict: dict[str, list[float] | None] = {}
-        embedded_chunks: list[IndexChunk] = []
-
```

Comment:
minichunker moved to chunker.py.

its now an intermediate step between chunking and embedding